File: 백준/Bronze/34935. 오름차순과 비내림차순/오름차순과 비내림차순.cc
#include <iostream>
#include <vector>
using namespace std;
int main(void) {
    int n;
    bool rlt=true;
    cin>>n;
    vector<long long> arr(n);
    cin>>arr[0];
    for(int i=1;i<n;i++) {
        cin>>arr[i];
        if(arr[i-1]>=arr[i]) {
            rlt=false;
            break;
        }
    }
    cout<<rlt?1:0;
    return 0;
}

File: 백준/Bronze/34935. 오름차순과 비내림차순/오름차순과 비내림차순.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int n = scan.nextInt();
        boolean rlt=true;
        long[] arr = new long[n];
        arr[0]=scan.nextLong();
        for(int i=1;i<n;i++) {
            arr[i]=scan.nextLong();
            if(arr[i-1]>=arr[i]) {
                rlt=false;
                break;
            }
        }
        System.out.print(rlt?1:0);
    }
}